### DISCUSSION TITLE
Adding Bengali translation

### DISCUSSION THREAD

**Discussion by shamsch** [Syknapse/Contribute-To-This-Project#1440] [Q&A] [answered]
Hi, I would like to add Bengali translation. Is it okay if I just create a `BENGALI.md` file in the `./translations/` directory and do a pull request? 

Thanks.

**Comment by 9662e103-129a** [comment]
Hello @shamsch, if there is not already one, yup create the readme file and submit a PR and I will look through the PR (to make sure there are no conflicts)

**Comment by Preet-Sojitra** [comment, marked as answer]
> Hi, I would like to add Bengali translation. Is it okay if I just create a `BENGALI.md` file in the `./translations/` directory and do a pull request?
> 
> Thanks.

and do link your translation in `Readme.md`

**Reply by 9662e103-129a** [reply]
Also quick note @shamsch, if you are not sure how to add the link/format it to the README.md file, ping me and i'll add it

**Comment by cardosource** [comment]
It would be good for this project to have translation into other languages ​​of different nationalities?

**Reply by Preet-Sojitra** [reply]
> It would be good for this project to have translation into other languages ​​of different nationalities?

Hello, Translations in different languages are always welcome.
